src/test/java/com/vanniktech/dependency/graph/generator/DependencyGraphGeneratorLinkTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.vanniktech.dependency.graph.generator
+
+import com.vanniktech.dependency.graph.generator.DependencyContainer.Project
+import com.vanniktech.dependency.graph.generator.DependencyContainer.ResolvedDependency
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorExtension.Generator.Companion.ALL
+import guru.nidi.graphviz.attribute.Color.BLUE
+import guru.nidi.graphviz.attribute.Style.DASHED
+import guru.nidi.graphviz.model.Link
+import org.gradle.api.internal.project.DefaultProject
+import org.gradle.api.plugins.JavaLibraryPlugin
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class DependencyGraphGeneratorLinkTest {
+
+  private lateinit var project: DefaultProject
+
+  @Before
```

Comment:
nit: do you mind placing the annotation on the same line as the fun methods?

So that it is `@Before fun setUp() {`

Reply:
Yes, will do!